packages/offix-client/integration_test/test/conflict.test.js

```diff
@@ -2,15 +2,18 @@ import { ApolloOfflineClient } from '../../dist';
 import { TestStore } from '../utils/testStore';
 import { ToggleableNetworkStatus } from '../utils/network';
 import { InMemoryCache } from "apollo-cache-inmemory";
-import server from '../utils/server';
-import {
-  ADD_TASK,
-  GET_TASKS,
-  UPDATE_TASK_CLIENT_RESOLUTION,
-} from '../utils/graphql.queries';
 import { HttpLink } from 'apollo-link-http';
+import { TestxDirector } from "graphql-testx/dist/src/TestxDirector"
```

Comment:
This might backfire. Can we export that in next version?

Reply:
I have created an issue in testx to solve this porblem: https://github.com/aerogear/graphql-testx/issues/118